scripts/setup/setup.sh

```diff
@@ -22,15 +25,17 @@ run_scripts() {
     readonly target_dir
     shift
 
+    set +e
     for script in "${target_dir}"/*.sh; do
         # TODO: Add name script name 
-        # TODO: 
+        # TODO: Test whether the script exiting with non-zero status will be caught by set -e (We don't want this to happen)
```

Comment:
I thought this is the behavior we wanted: non-zero status causes the script to exit.

Reply:
Well, we want the error message to print first in the setup.sh script since the behavior is still exiting if the status of the child script is non-zero. Let me know if I am misinterpreting the code/behavior.